# contracts/test/FHERC20_Harness.sol  ->  contracts/test/FHERC20_Harness.fsol
# upstream 5138cb8 .. ported 4cefd22

diff --git a/contracts/test/FHERC20_Harness.sol b/contracts/test/FHERC20_Harness.fsol
index 3be8c5b..6587e93 100644
--- a/contracts/test/FHERC20_Harness.sol
+++ b/contracts/test/FHERC20_Harness.fsol
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.25;
 
 import { FHE, euint64 } from "@fhenixprotocol/cofhe-contracts/FHE.sol";
-import { FHERC20 } from "../FHERC20/FHERC20.sol";
+import { FHERC20 } from "../FHERC20/FHERC20.fsol";
 
 contract FHERC20_Harness is FHERC20 {
     constructor(
@@ -13,10 +13,10 @@ contract FHERC20_Harness is FHERC20 {
     ) FHERC20(name_, symbol_, decimals_, contractURI_) {}
 
     function mint(address account, uint64 value) public {
-        _mint(account, FHE.asEuint64(value));
+        _mint(account, euint64(value));
     }
 
     function burn(address account, uint64 value) public {
-        _burn(account, FHE.asEuint64(value));
+        _burn(account, euint64(value));
     }
 }
